### PR TITLE
Update sign in and sign up screens. Add sign in to the header.

### PR DESCRIPTION
**Pull request by andrewmaier**

### CODE REVIEW

**Comment by andrewmaier** [comment]
Moved this card to "acceptance testing," @DavidEBest: 
https://trello.com/c/qsmQ1oGG/102-fix-sign-in-page

File: sbirez/static/sass/main.scss [hunk 1]
```diff
@@ -1902,10 +1852,6 @@ body .bigfoot-footnote__button:after {
   clear: both;
 }
 
-.bigfoot-footnote__wrapper {
```
Comment (DavidEBest) [inline sbirez/static/sass/main.scss]: Was this removed on purpose?

File: sbirez/static/sass/main.scss [hunk 2]
```diff
@@ -1504,34 +1433,6 @@ body.proposals-no-results main div.wrap {
 body.proposals-no-results main div.wrap h1,
 body.topics-no-results main div.wrap h1{ margin-top: 0; }
 
-body.proposals-no-results main form {
```
Comment (DavidEBest) [inline sbirez/static/sass/main.scss]: Were these removed on purpose?

**Comment by andrewmaier** [comment]
Egads! They were not. Did that break the footnotes?

**Comment by andrewmaier** [comment]
Looks good to me @DavidEBest !

**Pull request merged**